### ISSUE TITLE
Spec file

### ISSUE THREAD

**Issue by d4s** [dkukharau/simplehuffman#1]
- Version -- bad idea to use timestamp as version. In case of time screw you may break upgrade.
  Please read carefully: https://fedoraproject.org/wiki/Packaging:NamingGuidelines#Package_Versioning
- Changelog -- must use the same %version-%release as you have in spec.

**Comment by dkukharau** [comment]
@d4s
Yeah, I agree. I changed spec file to use the following format for versioning : 
{major_version}-{release_number}.{git_commit_hash}, and increment release_number on each build
see 7adaebf171f37dfeda485ed44b601c3b4363622c
example snapshot name: simplehuffman-1.0-4.81ad7c3.x86_64.rpm

As for changelog, there are some problems to update it on each build, because ones are triggered automatically. Every time a build is triggered we'll have to update %changelog because %release is changed. This must be done from CI server, which is not a good idea, I guess.
Maybe it would be better just not to use it at all? I don't see much sense in it for now.
